tests/build.rs

```diff
@@ -0,0 +1,24 @@
+use std::io::Write;
+
+fn main() {
```

Comment:
I glanced at crates.io/crates/test-case and others, but this was more straightforward

Reply:
I'm glad it turned out this way.